CesiumGeometry/include/CesiumGeometry/AxisAlignedBox.h

```diff
@@ -120,6 +123,30 @@ struct CESIUMGEOMETRY_API AxisAlignedBox final {
            position.y >= this->minimumY && position.y <= this->maximumY &&
            position.z >= this->minimumZ && position.z <= this->maximumZ;
   }
+
+  /**
+   * @brief Creates a tight-fitting, axis-aligned bounding box that contains all
+   * of the input positions.
+   *
+   * @param positions The positions.
+   * @returns An axis-aligned bounding box derived from the input positions.
+   */
+  static AxisAlignedBox
+  fromPositions(const std::vector<glm::dvec3>& positions) {
```

Comment:
Please create an AxisAlignedBox.cpp and put this implementation there. Less code in headers means faster compile times! This class hasn't needed a .cpp so far because all of its methods are constexpr.

Reply:
Makes sense. I noticed the lack of .cpp but didn't realize the reason 😅